src/MongoDB/CursorId.stub.php

```diff
@@ -7,6 +7,7 @@
 
 namespace MongoDB\Driver;
 
+/** @deprecated deprecated without replacement */
```

Comment:
Good point. Looking into `zend_compile.h`, `ZEND_ACC_DEPRECATED` shares a value with `ZEND_ACC_PROTECTED_SET` (used as property flag only for asymmetric visibility), and `ZEND_ACC_USE_GUARDS` (set on the class when any `__get`, `__set`, `__isset`, or `__unset` are implemented by a class). Since `ZEND_ACC_DEPRECATED` is defined as a function flag and its value conflicts with a class flag with an entirely different meaning, I've removed the `@deprecated` comment. This seemed easier than teaching `gen_stub.php` that `@deprecated` on a class should not result in `ZEND_ACC_DEPRECATED` being added to `ce_flags`. I'll still follow up on whether gen_stub.php should be changed to account for this.